python/gen_requirements.py

```diff
@@ -277,7 +277,7 @@
     ("torch", None),
     ("torchvision", None),
     ("tornado", None),
-    ("xgboost", ">=1.1.0"),  # From PR #4953.
+    ("xgboost", "==1.5.0"),  # latest xgboost is not avaliable. Version mismatch will be fixed later.
```

Comment:
Ditto, you may preserve th previous comments
```suggestion
    ("xgboost", "'>=1.1.0,<1.6.0'"),  # From PR #4953 & Issue #12009
```